Vagrantfile

```diff
@@ -7,18 +62,76 @@ Vagrant.configure("2") do |config|
     end
     v.memory = 4096
   end
- [
-   %w{centos6.5 chef/centos-6.5},
-   %w{centos7   chef/centos-7.0},
-   %w{ubuntu14  ubuntu/trusty64},
-   %w{ubuntu12  ubuntu/precise64},
-   %w{freebsd10 chef/freebsd-10.0}
- ].each do |machine|
-  (name, box) = machine
+
+  config.vm.provider :aws do |aws, override|
+
+    # Required. Credentials for aws api. vagrant-aws will error if these are unset.
+    aws.access_key_id = ENV['AWS_ACCESS_KEY_ID']
+    aws.secret_access_key = ENV['AWS_SECRET_ACCESS_KEY']
```

Comment:
yes, i've tested this with the local centos7, centos6.5, ubuntu12, ubuntu14, and freebsd10 targets.